modules/compliance-operator-hcp-install.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-installation.adoc
+
+:_content-type: PROCEDURE
+[id="installing-compliance-operator-hcp_{context}"]
+= Installing the Compliance Operator on Hosted Control Planes
+
+The Compliance Operator can be installed in Hosted Control Planes using the OperatorHub by creating a Subscription file.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Define a `Subscription` object:
```

Comment:
I think they would also need steps 1-4 from https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-installation.html, I wonder if we should put this doc next to that one, or link the installation guide to this one.

Reply:
Gotcha - I thought we were creating the Subscription so that OperatorHub could be used for the rest of installation. I'll copy over the steps from the CLI install :+1: 

Do we still need the OperatorHub install instructions at the end of the procedure?

Reply:
I think this version looks good now!